tests/browser/contacts_accounts/merge_contacts.robot

```diff
@@ -0,0 +1,44 @@
+*** Settings ***
+
+Resource        tests/NPSP.robot
+Suite Setup     Open Test Browser
+Suite Teardown  Delete Records and Close Browser
+
+*** Test Cases ***
+
+Merge Contacts
+    # Manually creating contact records, sosl doesn't always return api generated contacts
+    Go To Object Home  Contact
+    Click Link  title:New
+    Populate Form
+    ...                       First Name=Robot
+    ...                       Last Name=One
+    Click Modal Button        Save & New
+    Populate Form
+    ...                       First Name=Robot
+    ...                       Last Name=Two
+    Click Modal Button        Save
+    ${winning_id} =  Get Current Record Id
+    Store Session Record  Contact  ${winning_id}
+    Go To Object Home  Contact
+    Page Should Contain  Robot One
+    Page Should Contain  Robot Two
+    Go To Setup Home
+    Open App Launcher
+    Input Text  //input[@placeholder='Search apps or items...']  Contact Merge
+    Click Link  Contact Merge
+    Select Frame with ID  vfFrameId
+    Input Text  //input[@placeholder='Search Contacts']  Robot
+    Click Element  //input[@value='Search']
+    Click Element  (//span[@id='fauxCBSelect'])[1]
+    Click Element  (//span[@id='fauxCBSelect'])[2]
+    Click Element  //input[@value='Next']
+    Click Button  name:Merge
```

Comment:
I had to change this one to:

        Click Element  (//button[text()='Merge'])[1]

Reply:
Use existing keyword 'Click Managehh Button__Merge'

Reply:
@davisagli - assuming you had to change it because the current version didn't work for you? It's working for me the way it is, so I'm curious what / why it wasn't working for you.

Reply:
There isn't a button with `name="Merge"`. You can see the log of the failed MetaCI build here: https://mrbelvedereci.herokuapp.com/tests/result/32058172

Reply:
Thanks, I just realized I never actually checked the robot tests in MetaCI!